app/controlplane/internal/data/ent/migrate/migrations/20240610060712.sql

```diff
@@ -0,0 +1,6 @@
+-- Add default value to "name" column in "workflows" table if it is null
+UPDATE "workflows" SET "name" = CONCAT(split_part("id"::text, '-', 1)) WHERE "name" IS NULL;
```

Comment:
Only setting the default vale as part of the ID. Originally I used a concatenation of project + id but the project might be null as well although neither on the CLI and UI is optional, I wanted to play safe here.

Reply:
How did you create the migration? In theory no migration should have been created, I believe the constraints are already in place. Or am I missing something?

also, did that migration work for you? What happened when you ran the controlplane locally? Did it work?

Reply:
What I did was:
- Create changes on `app/controlplane/internal/data/ent/schema/workflow.go`
- Ran `make generate`
- Create an empty migration by `make migration_new`
- Wrote the changes you see on the file
- Run `make migration_sync`

I've ran the application and the migration worked without problem. But double checking the code it seems the index was already created but I couldn't see it on my schema, which is weird.

In any case, I'll remove those changes since they are not needed.

Reply:
thanks, let's figure out why your environment was different. 

Can you 

- go to `main`
- docker compose down (also removing the docker volumes)
- docker compose up.
- Load the psql console `docker compose exec postgresql psql -U postgres controlplane` 
- `d workflows`?

Reply:
Yes, I can see the index now, it might have been the outdated volume on the DB. Thanks Migue!